transmart-data/README.md

```diff
@@ -76,6 +77,9 @@ The several options are fairly self-explanatory.
 
 ### PostgreSQL-specific notes
 
+These scripts were tested with PostgreSQL version 9.5. PostgreSQL version 10.2 
```

Comment:
@mickmis  any details on what did not work?

Reply:
At least one thing, which is the creation of the database. The error is:
```bash
/Library/PostgreSQL/10/bin/psql -v ON_ERROR_STOP=1 --single-transaction -X -d template1 -c "CREATE DATABASE transmart TEMPLATE template0 \
      ENCODING='UTF-8' LC_COLLATE='en_US.UTF-8'"
ERROR:  CREATE DATABASE cannot run inside a transaction block
```
Because this is the very beginning of the loading script, others things could be broken with PostgreSQL 10.2.

Reply:
ah, sounds familiar. Its something to fix for the future releases.